### DISCUSSION TITLE
Floating Window Snapping

### DISCUSSION THREAD

**Discussion by GumbyXGames** [Archfx/duo-de#87] [Ideas]
Can you please be added or enabled in the ROM? I believe it is a feature in A15 but it could only be in the beta releases.

**Comment by Archfx** [comment]
Can you point me to any material related to this? I'm interested in the idea, couldn't find anything related to A15

**Reply by GumbyXGames** [reply]
I think Google AI was "hallucinating" and gave me bad info. I can't get the same AI result using the same search terms I did yesterday. 

There is window snapping in Desktop Mode (https://www.androidauthority.com/android-15-desktop-mode-demo-3430991/, https://m.gsmarena.com/google_is_working_on_enhanced_android_desktop_mode_for_android_15-news-62335.php ) but not sure if that can be implemented on the Duo.

Edit: Found search terms that consistently give the AI prompts I got (for now) : `"android" freeform window snap edge`

**Reply by Archfx** [reply]
Looks like we will have access to this feature with [`Android 15 QPR1 Beta 2`](https://android-developers.googleblog.com/2024/09/developer-preview-desktop-windowing-on-android-tablets.html) which should be available in a couple of months.
